src/Teams/Domain/ProgramTextModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using Teams.Domain;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace Teams.Models
+{
+    [Table("Queue")]
```

Comment:
Let's change requirement and use the same name for the table as for the appropriate domain object.